extension/main.js

```diff
@@ -90,16 +112,17 @@
 			ghTab.url += '/participating';
 		}
 
-		chrome.tabs.query({currentWindow: true, url: ghTab.url}, tabs => {
-			if (tabs.length > 0) {
-				ghTab.highlighted = true;
-				chrome.tabs.update(tabs[0].id, ghTab);
-			} else if (tab.url === 'chrome://newtab/') {
-				chrome.tabs.update(null, ghTab);
-			} else {
-				chrome.tabs.create(ghTab);
-			}
-		});
+		// request optional permissions the 1rst time
+		if (window.GitHubNotify.settings.get('optional_permissions') === undefined) {
+			chrome.permissions.request({
```

Comment:
Hum, are you sure about that? If I remember, the ```request``` method displays the permission pop-up each time.
I should test! :-)